Kconfig.nrf

```diff
@@ -121,6 +121,24 @@ config NRF_SECURITY_ENABLER
 	imply MBEDTLS_ENABLE_HEAP if BT_HOST_CRYPTO_PRNG || BT_ECC || BT_GATT_CACHING || BT_CRYPTO
 	depends on !SOC_NRF5340_CPUNET
 
+# Bluetooth TX processor thread stack Thread Analyzer measurements
```

Comment:
Side note: commit message states that `Applications can still override via prj.conf.`. Unless I missed something, both thread stack size and priority are promptless.

Also `konfig.nrf:` -> `Kconfig.nrf`